FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/TableKeyProvider.cs

```diff
@@ -13,6 +14,10 @@ public static class TableKeyProvider
 
         public static string TriggerRowKey(byte queueType) => $"{queueType:D3}_trigger";
 
+        public static string JobStatusPartitionKey(byte queueType, int jobType) => $"{queueType:D3}_{jobType}_jobStatus";
+
+        public static string JobStatusRowKey(byte queueType, int jobType, long groupId) => $"{queueType:D3}_{jobType}_{groupId:D20}";
```

Comment:
Shall we use job it in the RowKey so that we have ways to save status for processing jobs

Reply:
Added job id here.